models/biocro/R/model2netcdf.BIOCRO.R

```diff
@@ -78,7 +79,7 @@ model2netcdf.BIOCRO <- function(result, genus = NULL, outdir, lat = -9999, lon =
                        Evap = udunits2::ud.convert(SoilEvaporation + CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
                        TVeg = udunits2::ud.convert(CanopyTrans, "Mg/ha/h", "kg/m2/s"), 
                        LAI = LAI))
-    
+    RR$NPP <- udunits2::ud.convert(c(0, diff(R$TotLivBiom)), "kg/m2/h", "kg/m2/s")
```

Comment:
@infotroph thanks for pointing out the k multiplier -- given that it looks like a lot of the existing variables are in the wrong units (biomass not carbon). It also looks like this specific line will fail since RR$NPP is being calculated from R$TotLivBiom but that TotLivBiom is only in RR not R.